packages/cli/src/models/local/KitController.ts

```diff
@@ -0,0 +1,116 @@
+import path from 'path';
+import axios from 'axios';
+import fs from 'fs-extra';
+import Ajv from 'ajv';
+
+import stdout from '../../utils/stdout';
+import patch from '../../utils/patch';
+import child from '../../utils/child';
+import Spinner from '../../utils/spinner';
+
+const simpleGit = patch('simple-git/promise');
+
+export const DEFAULT_VERSION = '0.1.0';
+
+export interface KitConfig {
+  version: string;
+  message: string;
+  files: string[];
+  hooks: object;
+}
+
+/* tslint:disable */
+// schema has to have doulbe quotes
+const kitConfigSchema = {
+  "type": "object",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "properties": {
+    "version": { "type": "string" },
```

Comment:
Let's name the field differently (we use `zosversion` in `zos.json` files for instance, `manifestVersion` would also work), so we avoid confusion with a potential version of the kit itself.